site/cds_rdm/inspire_harvester/transform_entry.py

```diff
@@ -127,12 +135,39 @@ def _transform_titles(self):
 
         return rdm_title, rdm_additional_titles
 
+    def _validate_imprint(self):
+        """Validate that record has only 1 imprint."""
+        imprints = self.inspire_metadata.get("imprints", [])
+
+        if not imprints:
+            return {}
+        if len(imprints) > 1:
+            self.metadata_errors.append(
+                f"More than 1 imprint found. INSPIRE record id: {self.inspire_metadata.get('control_number')}."
+            )
+            return {}
```

Comment:
I think in the next PRs we should revise on what data structures we return and why. Returning an empty dict does not seem like a good practice